### PR TITLE
Migrate to GitHub Actions.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -4,10 +4,7 @@ source = gocept.bbissues
 
 [report]
 precision = 2
-omit = */interfaces.py
-       */interfaces/*
-       */tests.py
-       */tests/*
+fail_under = 44
```
Comment (sallner) [inline .coveragerc:7]: coveralls actually shows 45.5%.

**Review by sallner — APPROVED** [review]
Looks good, although the coverage really seems bad.

File: .coveragerc (hunk 1 above)
Comment (icemac) [reply .coveragerc:7]: Locally running `tox -ecoverage` I just get 44.83 %. I do not know what's different on coveralls.io. (But I had the same before vice versa for zopefoundation packages.)

**Comment by icemac** [comment]
> Looks good, although the coverage really seems bad.

This package is not tested at all. The few existing tests do not test any actual implementation.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**